pkg/convert/jfr/parser.go

```diff
@@ -3,19 +3,21 @@ package jfr
 import (
 	"context"
 	"fmt"
-	"io"
-
 	"github.com/hashicorp/go-multierror"
 	"github.com/pyroscope-io/jfr-parser/parser"
 	"github.com/pyroscope-io/pyroscope/pkg/storage/metadata"
+	"io"
+	"regexp"
 
 	"github.com/pyroscope-io/pyroscope/pkg/storage"
 	"github.com/pyroscope-io/pyroscope/pkg/storage/segment"
 	"github.com/pyroscope-io/pyroscope/pkg/storage/tree"
 )
 
 func ParseJFR(ctx context.Context, s storage.Putter, body io.Reader, pi *storage.PutInput, jfrLabels *LabelsSnapshot) (err error) {
```

Comment:
Do we need a flag to enable/disable merging behaviour?

Reply:
Let's merge like this and we can add it later if people ask for it.